build.sbt

```diff
@@ -0,0 +1,30 @@
+// shadow sbt-scalajs' crossProject and CrossType until Scala.js 1.0.0 is released
+import sbtcrossproject.{crossProject, CrossType}
+
+inThisBuild(Def.settings(
+  crossScalaVersions := Seq("2.10.7", "2.11.12", "2.12.6"),
+  scalaVersion := crossScalaVersions.value.last,
+  version := "0.1.0-SNAPSHOT",
+
+  scalacOptions ++= Seq(
+    "-deprecation",
+    "-feature",
+    "-encoding",
+    "utf-8",
+    "-Xfatal-warnings",
+  )
+))
+
+lazy val reflect = crossProject(JSPlatform, JVMPlatform)
+  .in(file("."))
+  .settings(
+    scalacOptions in (Compile, doc) -= "-Xfatal-warnings",
+  )
+  .jvmSettings(
+    compileOrder := CompileOrder.JavaThenScala,
+    libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
```

Comment:
Hahahaha, fair enough. I didn't realize that this would introduce a circular dependency between this project and uTest :P